demo-app/src/main/kotlin/io/getstream/video/android/DirectCallActivity.kt

```diff
@@ -54,6 +57,7 @@ class DirectCallActivity : ComponentActivity() {
 
     @Inject
     lateinit var dataStore: StreamUserDataStore
+    private var call: Call? = null
```

Comment:
I think a `lateinit var` is more appropiate here - then we don't need to have that many `!!` everywhere. Only in `onStop()` you can check for `::call.isInitialized`

Reply:
Great poing